src/libraries/Common/src/System/CodeDom/CodeTypeReference.cs

```diff
@@ -73,7 +73,7 @@ public CodeTypeReference(string typeName, CodeTypeReferenceOptions codeTypeRefer
             Initialize(typeName, codeTypeReferenceOption);
         }
 
-        public CodeTypeReference(string typeName)
+        public CodeTypeReference(string? typeName)
```

Comment:
nit: other constructor should probably also take nullable `typeName` - treat as optional since it's internal class

Reply:
I re-looked and it looks like `null` is never passed into this overload, so I removed `?` here.